pkg/kubelet/kuberuntime/kuberuntime_container_linux.go

```diff
@@ -298,3 +303,72 @@ func toKubeContainerResources(statusResources *runtimeapi.ContainerResources) *k
 	}
 	return cStatusResources
 }
+
+var isCgroup2UnifiedMode = func() bool {
+	return libcontainercgroups.IsCgroup2UnifiedMode()
```

Comment:
hm I'm not sure about this helper. I would personally advocate for a shorter import library alias instead

Reply:
Hey @haircommander! Thanks for your review!

The intention behind this is not to shorten the function call, but rather to be able to mock cgroup version while running unit tests. Please see https://github.com/kubernetes/kubernetes/pull/118764/commits/fe5ac1b56aecd864525edaa49ce7527282e8c20f. Without it, the unit tests are environment dependent, meaning that we can't test cgroups v1/v2 and instead test with the version that's installed on the environment that runs the test.

Does that make sense?